subprojects/core/src/main/java/org/gradle/api/services/internal/BuildServiceProviderNagger.java

```diff
@@ -50,7 +50,7 @@ private static boolean isServiceRequiredBy(TaskInternal task, BuildServiceProvid
 
     private static void nagAboutUndeclaredUsageOf(BuildServiceProvider<?, ?> provider, TaskInternal task) {
         deprecateBehaviour(undeclaredBuildServiceUsage(provider, task))
-            .withAdvice("Declare the association between the task and the build service using 'Task#usesService'.")
+            .withAdvice("Declare the association between the task and the build service using 'Task#usesService' - not doing this is deprecated.")
```

Comment:
I think this is the wrong place and the "is deprecated" message should be in the first string. 

We might be able to just say "This behavior is deprecated." in this case, I think it would look like:
> Build service 'counter' is being used by task ':broken' without the corresponding declaration via 'Task#usesService'. This behavior is deprecated.